include/mata/nfa.hh

```diff
@@ -917,17 +918,17 @@ inline bool is_universal(
 bool is_incl(
         const Nfa&         smaller,
         const Nfa&         bigger,
-        const Alphabet&    alphabet,
-        Word*              cex = nullptr,
+        Word*              cex,
```

Comment:
`cex` is still optional: There is an overloaded function without `cex`, calling this function with `cex = nullptr`. For the user, nothing changes (where `cex` is concerned). Only the internal logic changed. This is done so that the compiler is able to distinguish between a function call passing an alphabet as an argument and a function call passing `cex`.